lib/fog/storage/openstack/requests/put_object.rb

```diff
@@ -36,6 +36,15 @@ def put_object(container, object, data, options = {}, &block)
           request(params)
         end
       end
+
+      class Mock
+        def put_object(container, object, data, options = {}, &block)
```

Comment:
Swift returns the `etag` of the object on PUT. We could mimic similar behavior by returning an MD5 sum of the object. This would allow us to enable some of the requests tests that currently are skipped if `FOG_MOCK` is set.